distribution/docs/src/main/jdocs/content/_configuring/configuring-solr-appender-contents.adoc

```diff
@@ -0,0 +1,43 @@
+:title: Configuring Solr Appender
+:type: configuringAdminConsole
+:status: published
+:summary: Configuring Solr appender.
+:order: 19
+
+==== Solr Appender
+
+The Solr Appender is a Decanter collector which persists to Solr all events posted to `decanter/collect/*`.
```

Comment:
What is a Decanter collector? I see here https://karaf.apache.org/manual/decanter/latest-1/#_collectors that
>Decanter collectors harvest the monitoring data, and send this data to the Decanter appenders.

That doesn't sound like what this `SolrAppender` class does.